install/jakartaee/docs/ReleaseNotes-JakartaEETCK-9.0.html

```diff
@@ -0,0 +1,129 @@
+code {  color: #000099}
+h1 {  color: #000099}
+h2 {  color: #000099}
+h3 {  color: #000099; text-decoration: underline}
+h4 {  font-style: italic; color: #000099}
+.tabletitle {  font-style: italic; color: #ffffcc; font-size: small; font-variant: small-caps}
+.subtitle {  font-style: italic; color: #ffffcc; font-variant: small-caps}
+.emphasize  {  font-style: italic; color: #000099}
+.step  { font-weight: bold}
+-->
+</style>
+</head>
+<body bgcolor="#ffffdd" text="#000000" link="#0000ff" vlink="#840084" alink="#0000ff">
+
+<div align="center">
+<h1>Jakarta EE  9 Platform TCK<br>
+<em class="emphasize">Release Notes, October 2020</em></h1>
+</div>
+
+<hr>
+<table border="0" title="Note">
+ <tr valign="top">
+ <td><strong>Note:</strong></td>
+ <td><em>This is <strong>the initial version </strong> of the Jakarta EE 9 TCK. The Release Notes file may be updated via the Jakarta EE Platform Specification Web Site: <a href="https://jakarta.ee/specifications/platform/9/">https://jakarta.ee/specifications/platform/9/</a> contains the most current information about the Jakarta EE 9 Platform TCK, including any last minute changes. Always check that web-site for the most up-to-date information.</em></td>
+ </tr>
+</table>
+
+<hr>
+<h2>Jakarta EE 9 Platform  TCK Requirements, Facts, and Figures</h2>
+<blockquote>
+<p>The Jakarta EE 9 Platform  8 TCK requires a minimum of 3.7 GB of free disk space:</p>
+
+<ul>
+   <li>Jakarta EE 8 Platform  CTS (zipped): 502 MB</li>
+   <li>Jakarta EE 8 Platform  CTS (unzipped):  785 MB</li>
+   <li>Space for TCK log/report files: 2 GB minimum (trace off)</li>
+</ul>
+
+<p>The test suite bundle contains the following:</p>
+
+<ul>
+ <li>EARs: 2218</li>
+ <li>RARs: 14</li>
+ <li>JARs: 3689</li>
+ <li>WARs: 2347</li>
+ <p/>
+ <li>44,589 total tests:
+ <ul>
+    <li>Platform TCK : 42436</li>
+    <li>JAX-WS 3.0 TCK: 1703</li>
+    <li>SOAP 2.0 TCK : 450</li>
+	 </ul></li>
+</ul>
+
+<p>The tests for XML Webservices and SOAP are available as standalone TCKs and implementers are required to run these standalone TCKs if they provide their own implementations of these APIs that override and/or replace the implementations in the Java SE platform.</p>
+
+<p>Chapter 4, &quot;Installation&quot; in the <em>Jakarta EE 9 Platform TCK User's Guide</em>, explains how to install the Jakarta EE 9 Platform  TCK on Linux, and Windows systems.</p>
+</blockquote>
+
+<hr>
+<h2>These are the stand alone TCKs, porting bundles, and JDK used to test the Jakarta EE 9 Compatible Implementation</h2>
+<ul>
+  <li>JDK version 1.8.0_191</li>
+  <li>Jakarta EE 9 Platform CI, Eclipse GlassFish 6.0</li>
+  <li>CDI version 3.0 [porting bundle file name cdi-tck-glassfish-porting-3.0.0.zip]</li>
+  <li>Bean Validation version 3.0 [porting bundle file name bv-tck-glassfish-porting-3.0.0.zip]</li>
+  <li>Dependency Injection version 2.0 [porting bundle file name 330-tck-glassfish-porting-2.0.0.zip]</li>
+  <li>XML Binding version 3.0</li>
```

Comment:
The only required TCKs are:
- Jakarta Contexts and Dependency Injection
- Jakarta Bean Validation
- Jakarta Dependency Injection

Some EE 8 certification examples that show only these TCKs:
- GlassFish https://github.com/eclipse-ee4j/jakartaee-platform/issues/99
- WildFly https://github.com/eclipse-ee4j/jakartaee-platform/issues/121

We could list the other TCKs but they should be described as `optional`.